### PR TITLE
Adding Benchmarks back into agbench and updates to agbench

### PR DESCRIPTION
**Pull request by husseinmozannar**
## Why are these changes needed?

Added benchmarks back into autogen, notably GAIA, AssistantBench, HumanEval and WebArena. Renamed everything to be consistent with new package, enabled creation of endpoint from environment file.

Modified readme of magentic-one with a few changes regarding installation instructions

Fixed renaming issues in agbench to be consistent with new package structure.

### CODE REVIEW

**Comment by husseinmozannar** [comment]
@gagb  and @afourney  please review changes to agbench.

File: python/packages/agbench/dashboard_tabulate.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        </html>
+    """,
+        table_html=table_html,
+        summary_html=summary_html,
+        path=path,
+    )
+
+
+if __name__ == "__main__":
+    app.run(debug=True)
```
[the remark this answers is not in the document]
Comment (husseinmozannar) [reply python/packages/agbench/dashboard_tabulate.py]: removed the test app dashboard_tabulate

File: python/packages/agbench/dashboard_tabulate.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+import re
+
+app = Flask(__name__)
+
+
+def run_agbench_command(path):
+    command = f"agbench tabulate {path}"
+    result = subprocess.run(command, shell=True, capture_output=True, text=True)
+    command_csv = f"agbench tabulate {path} -c"
+    result_csv = subprocess.run(command_csv, shell=True, capture_output=True, text=True)
```
[the remark this answers is not in the document]
Comment (husseinmozannar) [reply python/packages/agbench/dashboard_tabulate.py]: removed app

**Review by gagb — APPROVED** [review]

**Comment by husseinmozannar** [comment]
should be good to merge!

**Pull request merged**